src/plugins/AngleMeasurementsPlugin/index.js

```diff
@@ -1,4 +1,61 @@
 export * from "./AngleMeasurementsPlugin.js";
 export * from "./AngleMeasurementsControl.js";
 export * from "./AngleMeasurementsMouseControl.js";
-export * from "./AngleMeasurementsTouchControl.js";
\ No newline at end of file
+export * from "./AngleMeasurementsTouchControl.js";
+
+import {Component} from "../../viewer/scene/Component.js";
+import {activateDraggableDots} from "../../../src/plugins/lib/ui/index.js";
+
+class AngleMeasurementEditControl extends Component {
+
+    /**
+     * Edits {@link AngleMeasurement} with mouse and/or touch input.
+     *
+     * @param {AngleMeasurement} [measurement] The AngleMeasurement to edit.
+     * @param [cfg] Configuration
+     * @param {PointerLens} [cfg.pointerLens] A PointerLens to use to provide a magnified view of the cursor.
+     * @param {boolean} [cfg.snapping] Whether to enable snap-to-vertex and snap-to-edge.
+     * @param {boolean} [handleMouseEvents] Whether to hangle mouse input.
+     * @param {boolean} [handleTouchEvents] Whether to hangle touch input.
+     */
+    constructor(measurement, cfg, handleMouseEvents, handleTouchEvents) {
```

Comment:
Could we please put the handleMouseEvents and handleTouchEvents into cfg

Reply:
Please note the `AngleMeasurementEditControl` is a "private" class, so its arguments are not at the moment part of the public API.
The public classes are `AngleMeasurementEditMouseControl` and `AngleMeasurementEditTouchControl`.
If `handleMouseEvents` and `handleTouchEvents` were to be properties of the `cfg` object, how should these properties be handled if already set inside the public classes' `cfg` argument.

Reply:
Sure, keep this as is, makes sense